kentikapi/client.go

```diff
@@ -109,8 +155,11 @@ func NewClient(c Config) *Client {
 		CloudExportAdminServiceAPI: cloudexportClient.CloudExportAdminServiceApi,
```

Comment:
Formatting seems to be off (likely on line 142, but GitHub does not allow to put a comment there).

Reply:
Formatting is ok (CI ensures that), but Github UI displays tabs with different width.